open_humans/templates/account/signup-form.html

```diff
@@ -11,21 +11,22 @@
   {% endif %}
 
   <div class="well">
-    <p>All members must have a <strong>public username and public
-      name</strong>.</p>
-
-    <p>
-      You can use any name, as long as it meets our
+    <p>Members must have a <strong>public username and public
+      name</strong>. You can use any name if it meets our
       <a href="{% url 'community_guidelines' %}#naming">Naming Guidelines</a>.
     </p>
 
     <p>
-      Your imported data will be private by default.
+      Imported data will be private by default. An account should be yours, or
+      someone for whom you are a legal guardian.
     </p>
+
     <p>
-      An account should represent an individual &ndash; your personal data, or
-      someone for whom you are a legal guardian.
+      This site is intended only for United States residents, but we
```

Comment:
Do intend to remain US-only indefinitely?  Otherwise, I do like the wording here.